src/main/kotlin/model/ManagePOMCases.kt

```diff
@@ -0,0 +1,161 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.*
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class ManagePOMCases private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var ldu: Person
+    lateinit var pom: Person
+    lateinit var homd: Person
+    lateinit var allocationManager: Container
+    lateinit var redis: Container
+    lateinit var activeJob: Container
+    lateinit var managePOMCasesAPI: Container
+    lateinit var db: Container
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "Manage POM Cases",
+        "A service for handling the handover of offenders from prison to probation"
+      )
+
+      ldu = model.addPerson(
+        "Local Delivery Unit",
+        "A Community/Probation team handling offenders who hail from a defined geographical area"
+      )
+
+      pom = model.addPerson(
+        "Prison Offender Manager (POM)",
+        "A prison staff member allocated to the prisoner"
+      )
+
+      homd = model.addPerson(
+        "Head Of Offender Management Delivery (HOMD)",
+        "A senior staff member who manages the prison's POM team"
+      )
+
+      allocationManager = system.addContainer(
+        "Offender Management Allocation Manager",
+        "Main application code - holds the UI and primary business logic",
+        "Ruby on Rails"
+      ).apply {
+        setUrl("https://github.com/ministryofjustice/offender-management-allocation-manager")
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      db = system.addContainer(
+        "MPC database",
+        "Database holding data mastered by MPC",
+        "AWS RDS PostgreSQL"
+      ).apply {
+          Tags.DATABASE.addTo(this)
+          CloudPlatform.rds.add(this)
+      }
+
+      managePOMCasesAPI = system.addContainer(
+        "manage POM Cases API",
+        "API exposing data mastered by the MPC Service",
+        "Ruby On Rails"
+      ).apply {
+        setUrl("https://moic.service.justice.gov.uk/api-docs/index.html")
```

Comment:
By convention at this time, these URLs point to the GitHub repo of the application -- could it point there too, please? We can make this better/more explicit later